app/models/UserAnswers.scala

```diff
@@ -67,14 +68,15 @@ final case class UserAnswers(
 
 object UserAnswers {
 
+  import MongoJavatimeFormats.Implicits._
   implicit lazy val reads: Reads[UserAnswers] = {
 
     import play.api.libs.functional.syntax._
 
     (
       (__ \ "_id").read[String] and
       (__ \ "data").read[JsObject] and
-      (__ \ "lastUpdated").read(MongoDateTimeFormats.localDateTimeRead)
```

Comment:
seems like this `MongoDateTimeFormats` is not used anywhere and can be deleted as class + spec